tests/recipe/test_amplitude_creation.py

```diff
@@ -33,7 +33,7 @@ def test_helicity(helicity_model: es.AmplitudeModel):
 
     builder = IntensityBuilder(model.particles, kinematics, phsp_sample)
     intensity = builder.create_intensity(model)
-    assert len(intensity.parameters) == 9
+    assert len(intensity.parameters) == 10
```

Comment:
why the parameters go up by 1?

Reply:
Because the meson radius is written as a parameter as well now

Reply:
Oh no, forget that: d2d1b90
This also came by in #159, I think

Reply:
This makes it a bit more transparent c666dd8